daisy_workflows/image_import/import_from_image.wf.json

```diff
@@ -17,6 +17,10 @@
       "Value": "true",
       "Description": "Whether to install GCE packages."
     },
+    "is_windows": {
```

Comment:
I don't 100% get the motivation. You have triggered CombineGuestOSFeatures in disk.go. Do you need to set this flag for "family" as well?

Reply:
I agree that there's a bit of overlap. Here are the differences:
 - Family is a free-form metadata field for users -- whatever they put in that field shows up in their web UI.
 - is_windows instructs daisy to add WINDOWS to GuestOsFeatures if it's not already there